lib/looker-sdk/client/dynamic.rb

```diff
@@ -84,6 +97,7 @@ def invoke(method_name, *args, &block)
 
       def method_missing(method_name, *args, &block)
         entry = find_entry(method_name) || (return super)
+        raise(NameError, "undefined method_name '#{method_name}'") unless entry
```

Comment:
how would it get here? note the early return in the line above.

Reply:
right, super throws on error, never returns.  Thanks!

Reply:
well, technically, `super` might find the called thing, for all we know at this layer. The point was that the early `return` ensures that the code below won't run regardless.

FWIW, an earlier iteration of this code had more going on and the early return seemed clean and concise to me at the time. Not it just looks concise :)